docs/CloudSpanner.md

```diff
@@ -0,0 +1,64 @@
+Trillian on Cloud Kubernetes with Cloud Spanner
+====================================================
+
+This document outlines the steps necessary to get an instance of Trillian up
+and running on Google Cloud.
+
+ 1. Create a Google Cloud project (e.g. trillian-opensource-ci)
+ 2. Enable APIs
+    1. Kubernetes
+    2. Cloud Spanner
+    3. ...
+
+ 3. Create the public API key (see https://cloud.google.com/kubernetes-engine/docs/tutorials/authenticating-to-cloud-platform)
+   1. From the APIs & Services > Credentials page, click Create credentials > API key.
+      A key will automatically be created.
+   2. Restrict API key to only have access to:
+      * Google Cloud APIs
+      * Cloud Spanner API
+   3. Download service account key (we'll call that file `service-key.json`)
+   4. run: `kubectl create secret generic spanner-key --from-file=key.json=service-key.json`
+
+ 4. Create Spanner instance & database
+   1. Click on menu > Cloud Spanner
+   2. Click on "Create Instance"
+   3. Use instance id "trillian-opensource-ci"
```

Comment:
Can we replace this with ${INSTANCE} or whatever throughout to make it easier?

Reply:
done